content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Membuat Pod Statis
+weight: 170
+content_type: task
+---
+
+<!-- overview -->
+
+Pod statis dikelola langsung oleh kubelet daemon pada suatu Node spesifik,
+tanpa {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+mengobservasi mereka.
+Tidak seperti Pod yang dikelola oleh control plane (contohnya,
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}});
+kubelet akan memantau setiap Pod statis (dan menjalankan ulang jika
+Pod mengalami kegagalan).
+
+Pod statis selalu terikat pada satu {{< glossary_tooltip term_id="kubelet" >}}
+pada Node spesifik.
+
+Kubelet secara otomatis akan mengulang untuk membuat sebuah
+{{< glossary_tooltip text="Pod mirror" term_id="mirror-pod" >}}
+pada Kubernetes API server untuk setiap Pod statis.
+Ini berarti Pod yang berjalan pada Node akan terlihat oleh API server,
+namun tidak dapat mengontrol darisana.
+
+{{< note >}}
+Jika kamu menjalankan Kubernetes terklaster dan menggunakan Pod statis
+untuk menjalankan Pod pada setiap Node, kamu kemungkinan harus menggunakan
+sebuah {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}.
+{{< /note >}}
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Halaman ini mengasumsikan kamu menggunakan {{< glossary_tooltip term_id="docker" >}}
+untuk menjalankan Pod, dan Node kamu bejalan menggunakan sistem operasi Fedora.
+Instruksi untuk distribusi lain atau instalasi Kubernetes mungkin beragam.
+
+<!-- steps -->
+
+## Membuat sebuah Pod statis
+
+Kamu dapan mengatur Pod statis dengan menggunakan sebuah
+[berkas konfigurasi pada _file system_](/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-filesystem)
+atau sebuah [berkas konfigurasi ditempatkan pada web](/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-http).
```

Comment:
for linking header in same page, do we need entire path? if so, we need to link it to indonesian one
```suggestion
[berkas konfigurasi pada _file system_](/id/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-filesystem)
atau sebuah [berkas konfigurasi ditempatkan pada web](/id/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-http).
```